src/script/widgets/ClassificationPanel.js

```diff
@@ -0,0 +1,317 @@
+    methodUniqueText: "Unique Values",
+    methodQuantileText: "Quantile",
+    methodEqualText: "Equal Intervals",
+    methodJenksText: "Jenks Natural Breaks",
+    standardDeviationText: "Standard Deviations",
+    attributeText: "Attribute",
+    selectAttributeText: "Select attribute",
+    startColor: "#FEE5D9",
+    endColor: "#A50F15",
+    generateRulesText: "Apply",
+    reverseColorsText: "Reverse colors",
+
+
+    initComponent: function() {
+        var colorFieldPlugins;
+        if (this.rulePanel.colorManager) {
+            colorFieldPlugins = [new this.rulePanel.colorManager];
+        }
+
+        var colorPanel =    new Ext.Panel({
+            hidden: true,
+            layout: 'form',
+            bodyStyle: {padding: "10px"},
+            border: false,
+            labelWidth: 70,
+            defaults: {
+                labelWidth: 70
+            },
+            items: [
+                {
+                    xtype: "gxp_colorfield",
+                    id: "choropleth_color_start",
+                    name: "color_start",
+                    fieldLabel: this.colorStartText,
+                    emptyText: OpenLayers.Renderer.defaultSymbolizer.strokeColor,
+                    value: this.startColor,
+                    defaultBackground: this.startColor,
+                    plugins: colorFieldPlugins,
+                    listeners: {
+                        valid: function(field) {
+                            this.rulePanel.rule[field.name] = field.getValue();
+                            //this.rulePanel.fireEvent("change", this.rulePanel, this.rulePanel.rule);
+                        },
+                        scope: this
+                    }
+                },
+                {   xtype: "gxp_colorfield",
+                    id: "choropleth_color_end",
+                    name: "color_end",
+                    fieldLabel: this.colorEndText,
+                    emptyText: OpenLayers.Renderer.defaultSymbolizer.strokeColor,
+                    value: this.endColor,
+                    defaultBackground: this.endColor,
+                    plugins: colorFieldPlugins,
+                    listeners: {
+                        valid: function(field) {
+                            this.rulePanel.rule[field.name] = field.getValue();
+                            //this.rulePanel.fireEvent("change", this.rulePanel, this.rulePanel.rule);
+                        },
+                        scope: this
+                    },
+                    scope: this
+                }
+
+            ]
+        });
+
+
+
+        var classNumSelector = new Ext.ux.form.SpinnerField({
+            fieldLabel: 'Classes',
+            id: "choropleth_classes",
+            minValue: 2,
+            name: 'intervals',
+            defaultValue: 5,
+            width: 110,
+            listeners: {
+                'change':function(spinner, value){
+                    this.rulePanel.rule['intervals'] = value;
```

Comment:
can you get the name from the spinner instead of hard-coding 'intervals'  again?